.goreleaser-nightly.yaml

```diff
@@ -0,0 +1,84 @@
+project_name: openfga-nightly
+
+before:
+  hooks:
+    - make install-tools
+    - go generate ./...
+
+builds:
+  - main: ./cmd/openfga
+    binary: openfga
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    ldflags:
+      - "-s -w"
```

Comment:
Please add the build info to the `ldflags` so that it's apparent what version of OpenFGA is running when developers pull the `nightly` image. At minimum the `commit` and `date` should be there. If you want to set the `version` just equal to 'nightly' then that's fine, because the commit and date should be sufficient to identify the nightly build for it.

```
ldflags:
  - "-s -w"
  - "-X github.com/openfga/openfga/internal/build.Version=v{{ .Version }}"
  - "-X github.com/openfga/openfga/internal/build.Commit={{.Commit}}"
  - "-X github.com/openfga/openfga/internal/build.Date={{.Date}}"
```

Reply:
Resolved in [`2fb6c78`](https://github.com/openfga/openfga/pull/508/commits/2fb6c78bb9703cf38416845979155b93c92bc761)